docs/running_an_mpc_node_in_tdx.md

```diff
@@ -0,0 +1,268 @@
+# Deploy an MPC node within a TEE
+
+This document describes the steps to deploy an MPC node within a TEE. Its
+expected audience is Near One engineers because it uses internal infrastructure.
+
+## Steps
+
+1. Provision a TDX server.
+2. Install dstack components and start core dstack services.
+3. Deploy an MPC node within dstack.
+4. Observe the public data for the MPC node
+
+### 1. Provision a TDX server
+
+TODO: Add some sentence here about providers.
```

Comment:
Not sure I understand this todo. I suggest removing it for now.

```suggestion
```

Alternatively write something like this:
```suggestion
To run the MPC node within a TEE, you need a TDX-enabled server. There are several providers to choose from, such as pheonixNap, OVHcloud, openmetal.io and Hivelocity to name a few.
```